### ISSUE TITLE
Kernel uses serialization runtime from itself, not the library

### ISSUE THREAD

**Issue by altavir** [Kotlin/kotlin-jupyter#161]
```kotlin
@file:DependsOn("org.jetbrains.kotlinx:kotlinx-serialization-core-jvm:1.1.0")
import kotlinx.serialization.builtins.*
Boolean.serializer().descriptor.isInline
```

`isInline` property is available in serialization 1.1.0, but Jupyter can't find it. In the current version (`0.8.3.275`) serialization, 1.0.1 is used so it probably takes serialization from there.

**Comment by altavir** [comment]
The latest build (`0.8.3.279`) Indeed fixes the problem, but it is only a workaround. Platform serialization should not leak into the notebook content.

**Comment by ileasile** [comment]
Problem should be fixed in `0.8.3.280`
See commit adaadf2bbedf67a4560b14b59269703b96226497
